src/ast.rs

```diff
@@ -147,7 +149,9 @@ pub struct Usize(pub u32);
 /// See <https://google.github.io/xls/dslx_reference/#bit-type>
 #[derive(Debug, PartialEq, Clone)]
 pub struct RawBitType {
+    /// a bit type is differentiated on if it's signed or unsigned
```

Comment:
Comment is less complete-sentence-y than others. Capitalize, punctuate, etc.

Reply:
done